scripts/hyperspectral/hyperspectral_workflow.sh

```diff
@@ -517,13 +518,13 @@ for ((fl_idx=0;fl_idx<${fl_nbr};fl_idx++)); do
 	typ_in_ENVI=$(grep '^data type' ${hdr_fl} | cut -d ' ' -f 4 | tr -d '\015')
 	xps_tm=$(grep 'current setting exposure' ${mtd_fl} | cut -d ':' -f 2 | tr -d '" ,\015' )
 	fl_clb="${drc_spt}/calibration_vnir_${xps_tm}ms.nc"
-	if [ "${wvl_nbr}" -eq 272 ]; then 
+	if [ "${wvl_nbr}" -eq 272 ] || [ "${wvl_nbr}" -eq 273 ]; then 
```

Comment:
this works for now ... but is there a way to generalize this so that it can be used with another hyperspectral sensor? We can wait until we actually have another sensor and want to generalize the workflow. 

But could `wvl_nbr` be queried from the sensor metadata?